contracts/SlotMachine.sol

```diff
@@ -21,7 +22,7 @@ contract SlotMachine {
     function play(uint256 _betAmount) public payable {
         // Uncomment below code once dummy user has balance
 
-        uint userBalance = msg.value ;
+        uint256 userBalance = msg.value * (10**18); 
```

Comment:
this is the wei conversion no?

Reply:
This WEI conversion is for the user balance. The WEI conversion we do in front end is for the bet amount. 
Although, I am not sure is the user balance is returned as ETH or WEI. This still needs testing.

Reply:
Got it